src/App/App.scss

```diff
@@ -490,6 +490,10 @@ button.App-connect-wallet:hover {
   font-weight: bold;
 }
 
+.Page-title img {
```

Comment:
maybe specify a class for this image, like Page-title-image?

I would avoid such general styles because it leads to non-obvious behavior